src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+
+  let totalCost = days * DAILY_RATE;
+
+  if (days >= LONG_TERM) {
+    totalCost -= LONG_TERM_DISCOUNT;
+  } else if (days >= MEDIUM_TERM) {
+    totalCost -= MEDIUM_TERM_DISCOUNT;
+  }
```

Comment:
Remove the `else` keyword to follow the best practice of using `if` with `return` to simplify the flow. Since you're modifying the `totalCost` and returning it at the end of the function, the `else` is unnecessary and can be omitted.